styles/angular-patternfly.css

```diff
@@ -0,0 +1,29 @@
+.donut-chart-pf {
+  width: 100%;
+  float: left;
+  padding-top: 10px;
+}
+
+.donut-tooltip-pf {
+  background: #434343;
+  color: #fff;
+  opacity: 0.9;
+  filter: alpha(opacity=90);
+  padding: 2px 6px;
+}
+
+.donut-title-big-pf {
+  font-family: "Open Sans", Helvetica, Arial, sans-serif;
+  font-size: 22px;
+  font-weight: normal;
```

Comment:
The rest looks good so after you have looked at this let me know and I can merge.